spec/filters/fixtures/test_certs/ca.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This cert expires in a couple of years, will this break tests or is curl -k the only relevant validation skip we need?

Reply:
Indeed it will break integration specs eventually, just like in ES output: https://github.com/logstash-plugins/logstash-output-elasticsearch/tree/v11.4.1/spec/fixtures/test_certs where it's copied from. 

We do not have a base-line setup on generating TLS material in tests and there isn't a good enough excuse to start looking into that atm (simply due having more pressing stuff to tackle). 
Hopefully the TLS setting unification effort will be a place to look into these...